CMakeLists.txt

```diff
@@ -260,3 +260,5 @@ set(USB_IDS_INSTALL_PATH ${CMAKE_INSTALL_DATAROOTDIR}/${USB_IDS_FILE})
 if(ENABLE_USB_IDS_DOWNLOAD AND NOT EXISTS "${CMAKE_STAGING_PREFIX}/${USB_IDS_INSTALL_PATH}")
     download_file("${USB_IDS_URL}" "${CMAKE_STAGING_PREFIX}/${USB_IDS_INSTALL_PATH}")
 endif()
+
+include(ca-update)
```

Comment:
We'd usually include this from the CMakeLists.txt file of the library that actually needs this, I think. In this case, that would be `Userland/Libraries/LibTLS`.

Reply:
Also, please name it similar to the other files (those use `_` instead of `-`) and give it a more expressive name (like `ca_certificates.cmake`).

Reply:
Both 👍